tests/e2e/api/transaction.test.ts

```diff
@@ -86,4 +87,52 @@ describe("transaction api", () => {
       expect(transaction).toStrictEqual(txn);
     });
   });
+
+  describe("long poll", () => {
+    let txn: TransactionResponse;
+    beforeAll(async () => {
+      const senderAccount = Account.generate();
+      await aptos.fundAccount({ accountAddress: senderAccount.accountAddress, amount: FUND_AMOUNT });
+      const bob = Account.generate();
+      const rawTxn = await aptos.transaction.build.simple({
+        sender: senderAccount.accountAddress,
+        data: {
+          function: "0x1::aptos_account::transfer",
+          functionArguments: [bob.accountAddress, new U64(10)],
+        },
+      });
+      const authenticator = aptos.transaction.sign({
+        signer: senderAccount,
+        transaction: rawTxn,
+      });
+      const response = await aptos.transaction.submit.simple({
+        transaction: rawTxn,
+        senderAuthenticator: authenticator,
+      });
+      txn = await longWaitForTransaction({ aptosConfig: aptos.config, transactionHash: response.hash });
+    });
+
+    test("it queries for transactions on the chain", async () => {
+      const transactions = await aptos.getTransactions();
+      expect(transactions.length).toBeGreaterThan(0);
+    });
+
+    test("it queries for transactions by version", async () => {
+      if (!("version" in txn)) {
+        throw new Error("Transaction is still pending!");
```

Comment:
I dont think we want to throw while testing

Reply:
removed